fixtures/american-dj/encore-lp12z-ip.json

```diff
@@ -0,0 +1,580 @@
+          "comment": "Square"
+        },
+        {
+          "dmxRange": [21, 40],
+          "type": "Maintenance",
+          "comment": "Linear"
+        },
+        {
+          "dmxRange": [41, 60],
+          "type": "Maintenance",
+          "comment": "Inv Squa"
+        },
+        {
+          "dmxRange": [61, 80],
+          "type": "Maintenance",
+          "comment": "S Curve"
+        },
+        {
+          "dmxRange": [81, 255],
+          "type": "NoFunction"
+        }
+      ]
+    },
+    "Special Functions": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "Maintenance",
+          "comment": "Default to unit setting"
+        },
+        {
+          "dmxRange": [16, 30],
+          "type": "Maintenance",
+          "parameter": 900,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [31, 45],
+          "type": "Maintenance",
+          "parameter": 1000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [46, 60],
+          "type": "Maintenance",
+          "parameter": 1100,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [61, 75],
+          "type": "Maintenance",
+          "parameter": 1200,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [76, 90],
+          "type": "Maintenance",
+          "parameter": 1300,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [91, 105],
+          "type": "Maintenance",
+          "parameter": 1400,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [106, 120],
+          "type": "Maintenance",
+          "parameter": 1500,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [121, 135],
+          "type": "Maintenance",
+          "parameter": 2500,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [136, 150],
+          "type": "Maintenance",
+          "parameter": 4000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [151, 165],
+          "type": "Maintenance",
+          "parameter": 5000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [166, 180],
+          "type": "Maintenance",
+          "parameter": 10000
+        },
+        {
+          "dmxRange": [181, 195],
+          "type": "Maintenance",
+          "parameter": 15000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [196, 210],
+          "type": "Maintenance",
+          "parameter": 20000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [211, 225],
+          "type": "Maintenance",
+          "parameter": 25000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [226, 229],
+          "type": "Maintenance",
+          "comment": "Enable Zoom Mode 1"
+        },
+        {
+          "dmxRange": [230, 233],
+          "type": "Maintenance",
+          "hold": "3s",
```

Comment:
Should the other capabilities in this channel also have a `hold` value?

Reply:
others are instant as I understood from the manual, but I don't have access to this fixture anymore so I can't test it